eng/Version.Details.xml

```diff
@@ -170,10 +170,12 @@
     <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-8.0.100" Version="8.0.0-alpha.1.23073.2">
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>ff2362091c3ce19e09606a7de927670ba5d1cb81</Sha>
+      <SourceBuild RepoName="emsdk" ManagedOnly="true" />
     </Dependency>
     <Dependency Name="Microsoft.NET.Workload.Emscripten.net7.Manifest-8.0.100" Version="8.0.0-alpha.1.23073.2">
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>ff2362091c3ce19e09606a7de927670ba5d1cb81</Sha>
+      <SourceBuild RepoName="emsdk" ManagedOnly="true" />
```

Comment:
Apologies but there is a bug - you can only include this attribute once per repo.
```suggestion
```